code/modules/organs/organ_external.dm

```diff
@@ -1133,7 +1133,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 	return !(status & (ORGAN_MUTATED|ORGAN_DEAD))
 
 /obj/item/organ/external/proc/is_malfunctioning()
-	return ((robotic >= ORGAN_ROBOT) && (brute_dam + burn_dam) >= 25 && prob(brute_dam + burn_dam))
+	return ((robotic >= ORGAN_ROBOT) && (brute_dam + burn_dam) >= min_broken_damage && prob(brute_dam + burn_dam)) //VOREStation Edit - Makes robotic limb damage scalable
```

Comment:
See previous comment, though this math here is different. You'd be raising some from 25 to 35, and lowering some from 25 to 15, etc.